frontend/src/NodesDashboard.jsx

```diff
@@ -1,15 +1,99 @@
 import React from "react";
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Typography from '@material-ui/core/Typography';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { loadNodes } from './data/nodes/actions';
+import { isErrored, isLoading, isNotLoaded } from './data/utils';
+import Table from './Table';
+
+const mapStateToProps = ({ nodes }) => ({
+    nodes,
+});
+
+const mapDispatchToProps = {
+    loadNodes,
+}
 
 class ShardsDashboard extends React.Component {
```

Comment:
We should probably rename this

Reply:
Any ideas? We could prob sit down and try to figure out all the elements so the naming makes sense.

Reply:
`NodesDashboard` maybe? Yeah now we're not on the hackday definitely worth reassessing where everything is.